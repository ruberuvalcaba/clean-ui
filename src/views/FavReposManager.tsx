import React from 'react'

const FavReposManager = (): JSX.Element => {
  return <div>HELLO F* N* WORLD</div>
}

export default FavReposManager
