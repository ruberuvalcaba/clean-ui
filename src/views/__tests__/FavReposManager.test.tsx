import React from 'react'
import renderer from 'react-test-renderer'
import FavReposManager from '../FavReposManager'

it('FavReposManager component renders correctly', () => {
  const component = renderer.create(<FavReposManager />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
