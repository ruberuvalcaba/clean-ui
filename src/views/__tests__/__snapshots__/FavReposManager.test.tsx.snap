// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavReposManager component renders correctly 1`] = `
<div>
  HELLO F* N* WORLD
</div>
`;
